README.md

```diff
@@ -1,13 +1,11 @@
 # CacheMoney
-
 CacheMoney is the only banking app you'll ever need! Open a savings account today, set up direct deposit and enjoy $225 on us! Depositing money is a snap, CacheMoney's secure lines make sure your money is safe and accounted for. Finally your financial present and future is at your finger tips with CacheMoney! Our React frontend ensures our app is beautiful and responsive on all devices, and our servers make sure your information is safe and secure at all times.
```

Comment:
Edit the elevator pitch to be more technical. We do not give $225 nor offer direct deposit. I'd like to know more about what the app actually does.